lua/nvim-treesitter/parsers.lua

```diff
@@ -111,6 +111,13 @@ list.java = {
   maintainers = {"@p00f"},
 }
 
+list.kotlin = {
+  install_info = {
+    url = "https://github.com/QthCN/tree-sitter-kotlin",
+    files = { "src/parser.c" },
```

Comment:
You want to be the maintainer?

Reply:
for the hightlight-queries and/or the parser? I don't have experiance any with writing languageparsers (yet)

Reply:
for the highlights.